src/kernel_config_manager.sh

```diff
@@ -562,14 +560,15 @@ function remove_config()
   basic_config_validations "$target" "$force" 'Remove' "$msg"
 
   cd "$dot_configs_dir" || exit_msg 'It was not possible to move to configs dir'
-  git rm "$configs_dir/$target" "$dot_configs_dir/$metadata_dir/$target" > /dev/null 2>&1
-  git commit -m "Removed $target config: $USER - $(date)" > /dev/null 2>&1
+  if ! rm "$configs_dir/$target" "$dot_configs_dir/$metadata_dir/$target"; then
```

Comment:
Could you move the `rm` command outside this `if`? Usually, `rm` can be dangerous, and it is better to make it easier to see in the code. Also, make sure that both directories exist.
Finally, for string concatenation, check:
https://kworkflow.org/content/codingstyle.html#string-concatenation